src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -7660,12 +7667,24 @@ namespace HVACUnitarySystem {
         OnOffAirFlowRatio = 1.0;
         CoilCoolHeatRat = 1.0;
 
+        // Call the series of components that simulate a Unitary System
+        if (UnitarySystem(UnitarySysNum).ATMixerExists) {
+            // There is an air terminal mixer
+            if (UnitarySystem(UnitarySysNum).ATMixerType == ATMixer_InletSide) { // if there is an inlet side air terminal mixer
+                                                                                 // set the primary air inlet mass flow rate
+                Node(UnitarySystem(UnitarySysNum).ATMixerPriNode).MassFlowRate =
+                    min(Node(UnitarySystem(UnitarySysNum).ATMixerPriNode).MassFlowRateMaxAvail,
+                        Node(UnitarySystem(UnitarySysNum).AirInNode).MassFlowRate);
+                // now calculate the the mixer outlet conditions (and the secondary air inlet flow rate)
+                // the mixer outlet flow rate has already been set above (it is the "inlet" node flow rate)
+                SingleDuct::SimATMixer(UnitarySystem(UnitarySysNum).ATMixerName, FirstHVACIteration, UnitarySystem(UnitarySysNum).ATMixerIndex);
+            }
+        }
+
         // CALL the series of components that simulate a Unitary System
         if (UnitarySystem(UnitarySysNum).FanExists && UnitarySystem(UnitarySysNum).FanPlace == BlowThru) {
             if (UnitarySystem(UnitarySysNum).FanType_Num == DataHVACGlobals::FanType_SystemModelObject) {
-
-                HVACFan::fanObjs[UnitarySystem(UnitarySysNum).FanIndex]->simulate(
-                    _, _, _, _, m_massFlow1, m_runTimeFraction1, m_massFlow2, m_runTimeFraction2, _);
+                HVACFan::fanObjs[UnitarySystem(UnitarySysNum).FanIndex]->simulate(_, _, _, _);
```

Comment:
For set point based control, the SetOnOffMassFlowRate function is not called and is where m_massFlow1, m_runTimeFraction1, etc. get set. Since these are missing (and are 0) they should not be included in the fan call.